tadrep/visuals.py

```diff
@@ -19,6 +23,17 @@
 CONTIG_HEAD_MODIFIER = 200                                                  # Modifier of contig head length
 
 
+def create_plots():
+    json_files = cfg.output_path.glob('*.json')
+    log.info('detectet json-files=%i, path=%s', len(json_files), cfg.output_path)
+    for file in json_files:
```

Comment:
Why multiple JSON files? All information of a project (therefore all plasmid related information) can be stored within a single file.

Reply:
currently detection module writes one json file per input draft genome (because of threading) if plasmids were found.

I can see the benefits of a single file, trying to implement it

Reply:
should be resolved, detected plasmids in all input draft genomes are getting combined into a single json file.
visualization module was also adjusted to detect this json and loop through